Firmware/ultralcd.cpp

```diff
@@ -6941,6 +6948,63 @@ void lcd_sdcard_menu()
   }
   MENU_END();
 }
+#ifdef TMC2130
+static void lcd_belttest_v()
+{
+    lcd_belttest();
+    menu_back_if_clicked();
+}
+void lcd_belttest_print(const char* msg, uint16_t X, uint16_t Y)
+{
+    lcd_clear();
+    lcd_printf_P(
+              _N(
+                 ESC_H(0,0)
```

Comment:
@DRracer If you merge this and also the lcd improvements changes you need to get rid of that escape code. It is not needed in this case as lcd_clear() will also set the cursor to 0,0. Just remove line 6962.